dev/com.ibm.ws.kernel.feature.core/src/com/ibm/ws/kernel/feature/internal/subsystem/FeatureDefinitionUtils.java

```diff
@@ -104,6 +107,7 @@
         final String featureName;
         final String symbolicName;
         final String shortName;
+        final Set<String> alternateNames;
```

Comment:
I suggest making this a `Collection<String>` to give more freedom in the implementation.  Likely an ArrayList would be more performant than requiring a heavy object like a Set here.

Reply:
done